### PR TITLE
Elide graphql javadoc warning fixes

### LINKED ISSUE #1887: Fix elide-graphql javadoc warnings
Fix the following javadoc build warnings in elide-graphql:

```
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/Entity.java:27: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/Entity.java:144: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/Environment.java:22: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLConversionUtils.java:82: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLConversionUtils.java:87:5: Cyclomatic Complexity is 17 (max allowed is 15). [CyclomaticComplexity]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLConversionUtils.java:451: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLRequestScope.java:30:12: More than 7 parameters (found 8). [ParameterNumber]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/ModelBuilder.java:71: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/MutableGraphQLInputObjectType.java:89: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:109: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:120: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:152: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:242: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:331: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:358: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:379: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:448: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:481: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java:507: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java:204: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java:228: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java:249: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java:474: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java:494: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java:514: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/test/java/example/Pseudonym.java:18: First sentence should end with a period. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/test/java/example/package-info.java:6:1: Missing a Javadoc comment. [JavadocStyle]
[WARN] /sd/workspace/src/github.com/yahoo/elide/elide-graphql/src/test/java/graphqlEndpointTestModels/Incident.java:13: First sentence should end with a period. [JavadocStyle]
```

### PR DESCRIPTION
**Pull request by vandanabhandari**
Resolves #1887 

## Description
Fixes the checkstyle warnings.

## Motivation and Context
Fixes the checkstyle warnings.

## How Has This Been Tested?
`$ mvn -pl elide-graphql clean install
.....
[INFO] --- maven-checkstyle-plugin:3.1.2:check (validate) @ elide-graphql ---
[INFO] Starting audit...
[WARN] /Users/veda/vamsi/projects/yahoo/elide/elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLRequestScope.java:30:12: More than 7 parameters (found 8). [ParameterNumber]
Audit done.
[INFO] You have 0 Checkstyle violations.
[INFO]
[INFO] Installing /Users/veda/vamsi/projects/yahoo/elide/elide-graphql/target/elide-graphql-3.8.1-sources.jar to /Users/veda/.m2/repository/com/yahoo/elide/elide-graphql/3.8.1/elide-graphql-3.8.1-sources.jar
[INFO] Installing /Users/veda/vamsi/projects/yahoo/elide/elide-graphql/target/elide-graphql-3.8.1-javadoc.jar to /Users/veda/.m2/repository/com/yahoo/elide/elide-graphql/3.8.1/elide-graphql-3.8.1-javadoc.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:19 min
[INFO] Finished at: 2021-03-22T17:18:22-04:00
[INFO] ------------------------------------------------------------------------`
## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: elide-graphql/pom.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <artifactId>elide-graphql</artifactId>
+    <version>3.8.1</version>
```
Comment (aklish) [inline elide-graphql/pom.xml:7]: Don't pin the version here.

File: elide-integration-tests/src/test/java/example/models/generics/package-info.java [hunk 2]
```diff
@@ -4,4 +4,9 @@
  * See LICENSE file in project root for terms.
  */
 
+/*
+ * Copyright 2015, Yahoo Inc.
```
Comment (aklish) [inline elide-integration-tests/src/test/java/example/models/generics/package-info.java:8]: Copyright exists above.

File: elide-integration-tests/src/test/java/example/models/versioned/package-info.java [hunk 3]
```diff
@@ -4,6 +4,11 @@
  * See LICENSE file in project root for terms.
  */
 
+/*
+ * Copyright 2015, Yahoo Inc.
```
Comment (aklish) [inline elide-integration-tests/src/test/java/example/models/versioned/package-info.java:8]: Copyright exists above.

**Review by aklish — CHANGES_REQUESTED** [review]

**Review by vandanabhandari — COMMENTED** [review]
Addressed the comments

File: elide-integration-tests/src/test/java/example/models/versioned/package-info.java (hunk 3 above)
Comment (vandanabhandari) [reply elide-integration-tests/src/test/java/example/models/versioned/package-info.java]: addressed and removed

File: elide-graphql/pom.xml (hunk 1 above)
Comment (vandanabhandari) [reply elide-graphql/pom.xml]: oops, sorry abt that. Fixed it

**Review by aklish — APPROVED** [review]

**Pull request merged**